R/XcmsExperiment.R

```diff
@@ -1228,7 +1232,7 @@ setMethod(
     function(object, method = c("all", "closest_rt", "closest_mz",
                                 "largest_tic", "largest_bpi"),
              msLevel = 2L, expandRt = 0, expandMz = 0, ppm = 0,
-             skipFilled = FALSE, peaks = character(),
+             skipFilled = FALSE, peaks = character(), peaksInfo = c("rt", "mz"),
```

Comment:
Regarding the columns name, I already have the helper function do something like that so the column names have `peaks_` in front of them. so it follow the previous pattern of `peaks_id`. Are you okay with that ? 

Yes i'm okay with changing the variable name, however, quick question. Because i will implement the same thing in `featureSpectra()`. Should it be the same variable ? in this case the name needs to be neutral (why i chose `peaksInfo`). So maybe `addColumns` ? and we make it clear that in the case of chrompeaks it will come from `chromPeaks()` and for feature it will come from `featureDefinitions()` ?

Reply:
Because I don't know au I feel about have 2 different variables in the case of `featureSpectra()` (e.g. one such as `addfeatureColumns` and `addDhromPeaksColumns`) It just sound overly complicated and if someone ask for `featureSpectra()` I would expect they will use the feature informations no ?

Reply:
hm, yes, I see the point of having just one parameter `addColumns` for both... the documentation then has to clearly say what this refers to in the case of `featureSpectra()` and what for `chromPeakSpectra()`. But before settling on one common parameter, please rethink if we don't need to add any `chromPeak`-related columns if calling `featureSpectra()` (`featureSpectra()` is calling `chromPeakSpectra()`, so it would be easy to pass an (additional) parameter along).

For the prefix - I would suggest to have a parameter that would allow changing that but that has a meaningful prefix. So e.g. `addColumnsPrefix = "chrom_peak_"` for `chromPeakSpectra()` and `addColumnsPrefix = "feature_"` for `featureSpectra()`. I think it would be important to use self-explaining spectra variable names so that it's clear what is meant if one has only the `Spectra` object. E.g. for `sps$chrom_peak_rt` it would be pretty clear that the rt reported there is from the chrom peak.

Reply:
I mean i don't really see anything against implementing adding columns from both chrompeaks and features definitions, just wanted to avoid having too many parameters/columns. but it could come in the future that someone needs both info so might as well implement it now, I will do that.

Regarding the prefix, is this something the user need to decide ?  again i'm a bit worried by the amount of parameters/ their length. featureSpectra would then have: 
addColumnsChrompeaks = , addColumnsChromPeaksPrefix =, addColumnsFeature = , addColumnsFeaturePrefix = ... 

I'll start writing the code for it and see how it looks

Reply:
we could also hard-code the prefixes for now and have parameters later if needed.